app/views/quizmaster/quizzes/answers.html.haml

```diff
@@ -0,0 +1,32 @@
+.big-box
+  %h1= @question.body
+  %h3= @question.answer
+  %hr/
+
+  .answers
+    - @question.answers.each do |answer|
+      %divr{id: "answer_#{answer.id}"}
+        = answer.body
+        %divr{id: "right_or_wrong_#{answer.id}", class: "answer_div"}
+          %divr{class: "correct_buttons"}
+            = link_to image_tag('x-box.png', class: 'square-button'), "/quizmaster/quiz/answers/mark/#{answer.id}?correct=false", id: "wrong", remote: true
+            = link_to image_tag('check-box.png', class: 'square-button'), "/quizmaster/quiz/answers/mark/#{answer.id}?correct=true", id: "right", remote: true
+
```

Comment:
As always I prefer if you use path helpers. You can still pass in params: `your_path(id: answer.id, correct: 'true')`. Have you tried that?